//package supermarket.protocol;
//更新排行榜数据

message SingleRankItem
{
    optional string name	= 1;					
		optional int64 uid		= 2;
		optional int32 num	= 3;	
		optional string uid_str = 4;
		optional string url = 5;
	optional int32 wave = 6[default=0];	// 0=没变化，1=名次上升，-1=名次下降
}

message MseRank
{
	optional int32 rank_type = 1;
	optional SingleRankItem rank_item = 2;
	optional int64 rank_version = 3;
    optional string rankname = 4 [default=""];
    optional int32 opernum = 5[default=0]; //0: add single 1:refalsh all
    optional int32 rank_num = 6;
}